lib/bundle/whalebrew_dumper.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Bundle
+  module WhalebrewDumper
+    module_function
+
+    def reset!
+      @images = nil
+    end
+
+    def images
+      return [] unless Bundle.whalebrew_installed?
+
+      @images ||= `whalebrew list 2>/dev/null`.split("\n")
+      @images.reject { |image| image.include? "COMMAND   IMAGE" }
+             .map { |image| image.sub(/\w*\s+/, "") }
```

Comment:
This is the only part of this PR that I'm not entirely happy with but 🤷‍♂ . The issue here is that the output is this:

```
$ whalebrew list
COMMAND   IMAGE
dig       whalebrew/dig
wget      whalebrew/wget
```

The first line ditches the headers (I'm going to PR upstream to have a flag to remove so we may be able to remove it later) and then the second replaces everything at the beginning up until the first chunk of whitespace to get rid of the column formatting. I looked at splitting on `\t` however it didn't seem consistent despite that [being the delimiter upstream](https://github.com/whalebrew/whalebrew/blob/master/cmd/list.go#L37). If there are options here to make this saner, I'm all ears!

Reply:
It'd be awesome if `whalebrew list` had a `--json[=v1]` option.

Reply:
That would be very nice! I think the intention is the match the `docker` output as closely as possible so unsure how asking for a JSON output would go down 😛 My PR to remove the header row is at whalebrew/whalebrew#99